### DISCUSSION TITLE
Experiment Plan: Reducing Overfitting in CHIME Caption Classification

### DISCUSSION THREAD

**Discussion by anish1206** [KathiraveluLab/DREAMS#65] [General]
Hey @pradeeban 
I’m planning to experiment with the current BERT fine-tuning model to produce a better text classifier that can be used in the DREAMS app (more reliable predictions on real user captions).

### Dataset
I’ll load the dataset directly from Hugging Face: ayusrjn/CHIME-recovery-framework. (as suggested in the current experiment's README)

### Plan
I’ll keep the same split + same metrics across runs, then change only one thing at a time for controlled comparison.
1. Baseline: As current EXP-001
2. Improve generalization
3. Try lighter model
<img width="544" height="445" alt="image" src="https://github.com/user-attachments/assets/d34705d2-2cc2-450d-9621-2e7c628cfae8" />

### Evaluation

- Metrics: validation + test accuracy, macro‑F1, weighted‑F1
- Visuals: confusion matrix + train/validation loss curves
The goal is to reduce train–val gap and improve real world robustness so it’s suitable for app usage.

### Where I’ll work

- New folder: `experiment1_chime_text_overfitting`
- Notebook: `EXP-001_overfitting_reduction.ipynb`

If this is aligning with the projects future growth, I’ll like to open a pr for it.
Thanks.

**Comment by pradeeban** [comment]
Sounds good!
